### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#150] [Features: On Hold]
Shortly, There may be complex examples who people may coincide and would like to find their own solution. So a wiki would be helpful.

**Comment by h3rald** [comment]
Here, done: https://github.com/h3rald/min/wiki 😊

It's open to everyone for editing now, feel free to contribute if you wish!

**Reply by ghost** [reply]
Thanks for wiki! 👍

**Reply by ghost** [reply]
Now there is waiting moment for adding somethings to wiki. I'll wait *true stable* release of min.
